docs/user-guide/creating-application/workflow/ci-pipeline.md

```diff
@@ -14,9 +13,30 @@ Click on `Continuous Integration` to configure CI Pipeline, a new window will ap
 | `Stages` | 1.**Pre-build Stages**- Scripts to be executed before building an image.   2.**Docker build Stages**- Provide a new argument and override an old argument in key-value pair.   3. **Docker build Stages**- Scripts to be executed after building image |
 | `Scan for vulnerabilities` | It will scan your image and find if any vulnerabilities present |
 
+
+After Clicking on `Continuous Integration` to configure CI Pipeline, a new window will appear. 
+
+![](../../../.gitbook/assets/ca-workflow-ci-basic.png)
+
+This is the basic modal to configure CI pipeline, You need to provide 2 inputs as given below:
+
+### 1. Source Type
+In source type, select between Automatic or Manual. Here if you select Automatic, the CI pipeline will be triggered automatically whenever you push any commit to your repository. But if you select Manual, then you have to trigger your CI pipeline every time manually through the console.
+
+### 2. Branch Name
+In this column, you have to provide your branch or tag regex where your code is present and the branch to want this CI pipeline to be linked with.
```

Comment:
In this column, you have to provide the branch or tag name for which you would want CI pipeline to be executed. It supports both exact name as well as regex for both branch and tag.